libs/features/cdt/cdt.css

```
@@ -0,0 +1,86 @@
+.horizontal,
+.vertical {
+  display: flex;
+  padding: 20px 0;
+}
+
+.vertical {
+  flex-direction: column;
+}
+
+.center {
+  align-items: center;
+  justify-content: center;
+}
+
+.timer-label {
+  font-size: var(--type-body-s-size);
+  font-weight: 700;
+  height: 27px;
+}
+
+.light .timer-label {
```

given we only have light and dark, maybe we can do
```suggestion
.timer-label { // implies that light is the default
```